framework/rbac/AuthorizerInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace yii\rbac;
+
+/**
+ * @author Sam Mousa <sam@mousa.nl>
+ * @since 2.0.9
+ */
+interface AuthorizerInterface
```

Comment:
I like `checkAccessInterface`
On Jun 7, 2016 3:06 PM, "Bizley" <notifications@github.com> wrote:

> In framework/rbac/AuthorizerInterface.php
> <https://github.com/yiisoft/yii2/pull/11710#discussion_r66066198>:
>
> > @@ -0,0 +1,22 @@
> > +<?php
> > +namespace yii\rbac;
> > +
> > +/**
> > + * @author Sam Mousa <sam@mousa.nl>
> > + * @since 2.0.9
> > + */
> > +interface AuthorizerInterface
>
> Maybe JanitorInterface?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/yiisoft/yii2/pull/11710/files/e12e3e421b94c2455132d6f22874fa7350720fed#r66066198>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAhYzXy4swdklXCObWjbeQLcGSHhK9g7ks5qJWzkgaJpZM4IvrS_>
> .
>